From 6a3b7b28f34c91c72d8b849903d93de2997b82f1 Mon Sep 17 00:00:00 2001
From: Joseph Kogut <joseph.kogut@gmail.com>
Date: Tue, 8 Sep 2020 11:40:01 -0700
Subject: [PATCH] sentry.h: include ucontext.h

This is included by signal.h in newer versions of glibc, but results
in an unknown type name error in older versions.

Signed-off-by: Joseph Kogut <joseph.kogut@gmail.com>
---
 include/sentry.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/include/sentry.h b/include/sentry.h
index 25754d9..a55cc00 100644
--- a/include/sentry.h
+++ b/include/sentry.h
@@ -78,6 +78,7 @@ extern "C" {
 #    include <wtypes.h>
 #else
 #    include <signal.h>
+#    include <ucontext.h>
 #endif

 /**
--
2.28.0
